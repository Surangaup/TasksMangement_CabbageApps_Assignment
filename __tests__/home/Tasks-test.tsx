import * as React from 'react';
import renderer from 'react-test-renderer';
import Tasks from '../../src/screens/home/tasks';

test(('Tasks snapshot'), ()=>{
    const snap = renderer.create(<Tasks onCheckChange={()=>{}} tasks={[]} />).toJSON();
    expect(snap).toMatchSnapshot();
})