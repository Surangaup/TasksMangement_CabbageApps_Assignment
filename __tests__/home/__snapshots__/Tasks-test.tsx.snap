// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tasks snapshot 1`] = `
<RCTScrollView
  ListEmptyComponent={
    <ListEmptyComponent
      message="No task found"
    />
  }
  data={
    Array [
      Object {
        "data": Array [],
        "title": "Tasks",
      },
      Object {
        "data": Array [],
        "title": "Completed",
      },
    ]
  }
  disableVirtualization={false}
  getItem={[Function]}
  getItemCount={[Function]}
  horizontal={false}
  initialNumToRender={10}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  onContentSizeChange={[Function]}
  onEndReachedThreshold={2}
  onLayout={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={
    Array [
      0,
      2,
    ]
  }
  updateCellsBatchingPeriod={50}
  windowSize={21}
>
  <View>
    <View
      onLayout={[Function]}
      style={null}
    >
      <Text
        style={
          Object {
            "color": "#736F81",
            "fontSize": 16,
            "fontWeight": "bold",
            "marginLeft": 10,
          }
        }
      >
        Tasks
      </Text>
    </View>
    <View
      onLayout={[Function]}
      style={null}
    />
    <View
      onLayout={[Function]}
      style={null}
    >
      <Text
        style={
          Object {
            "color": "#736F81",
            "fontSize": 16,
            "fontWeight": "bold",
            "marginLeft": 10,
          }
        }
      >
        Completed
      </Text>
    </View>
    <View
      onLayout={[Function]}
      style={null}
    />
  </View>
</RCTScrollView>
`;
